packages/nimble-components/src/rich-text-editor/styles.ts

```diff
@@ -1,12 +1,159 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderHoverColor,
+    borderRgbPartialColor,
+    borderWidth,
+    smallDelay,
+    standardPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        inline-size: auto;
+        block-size: 100%;
```

Comment:
Is there a reason block-size and inline-size are being used instead of width and height?

The goal of those features is to be able to configure different writing modes but if you do that the widget does not render well

i.e. using `writing-mode: vertical-lr;`
![image](https://github.com/ni/nimble/assets/1588923/b3723971-03f9-47a4-bbed-49ccd61a49b9)

While I appreciate starting to think about it, Nimble does not use inline-size and block-size anywhere else and this implementation of support for different writing modes is incomplete and untested.

I think right now for maintainability (what most nimble / NI devs are familiar with) it would be better to use width and height until other writing modes can be properly tested, thoughts?

Reply:
Makes sense @rajsite. Agreed on the implementation and testing considerations, we need to take additionally for this.

Reply:
I have modified all the inline-size and block-size to width and height respectively.